charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.4.0
+version: 1.4.1
```

Comment:
Can you increase the version here? The version has already been updated to 1.4.1 in the meantime.

Reply:
shoot

Reply:
will do